funnel/views/organization.py

```diff
@@ -49,6 +51,17 @@ def people_and_teams(obj):
     return result
 
 
+@Organization.views('avatar_color_code', cached_property=True)
+def avatar_color_code(obj):
+    # Return an int from 1 to avatar_color_code from the initials of the given string
+    initial = initials(obj.title)
+    parts = initial.split()
+    string_total = ord(parts[0][0])
+    if len(parts) > 1:
+        string_total += ord(parts[0][1])
+    return string_total % no_avatar_colours or no_avatar_colours
+
+
```

Comment:
This is not required as the version in account.py with multiple decorators handles it.